File: Submissions/T07/E3.py
nomor = int(input("Masukkan nomor antrian : "))

if nomor % 2 == 0:
    print(f"Nomor {nomor} adalah : GENAP → Loket B")
else:
    print(f"Nomor {nomor} adalah : GANJIL → Loket A")

File: Submissions/T07/E3.cpp
#include <stdio.h>

int main() {
    int antrian;

    printf("Masukkan nomor antrian: ");
    scanf("%d", &antrian);

    if(antrian % 2 == 0) {
        printf("Nomor %d adalah : GENAP -> Loket B", antrian);
    } else {
        printf("Nomor %d adalah : GANJIL -> Loket A", antrian);
    }
    
    return 0;
}